docs/source/reference-core.rst

```diff
@@ -1096,6 +1096,8 @@ Broadcasting an event with :class:`Event`
 .. autoclass:: Event
    :members:
 
+.. autoclass:: EventStatistics
```

Comment:
Instead of documenting the class itself here, it might be possible to just put an [`index`](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#index-generating-markup) directive in the `statistics()` method, so the name links back there. That way we don't have the same information repeated twice.

Reply:
After trying a bunch of different variants I can't get `index` to create a `class`-type reference. I can create an index named `trio.EventStatistics` that I can link to with [`:ref:`](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#cross-referencing-arbitrary-locations) - but I failed to create one I can link to with `:class:` which is what autodoc generates the docstring to do.